docs/platform_management_plan/change_management.rst

```diff
@@ -12,14 +12,259 @@
    # SPDX-License-Identifier: Apache-2.0
    # *******************************************************************************
 
-Change management
-------------------------
+.. document:: Change Management Plan
+   :id: doc__platform_change_management_plan
+   :status: draft
+   :safety: ASIL_B
+   :tags: platform_management
+
+Change management / Change Management Plan
+------------------------------------------
+
+This document implements parts of the :need:`wp__platform_mgmt`.
 
 Purpose
 +++++++
+The purpose of the Change Management Plan is to guide the execution of the
+Change Requests of a project including their creation, analysis, approval, implementation,
+and verification.
+
 
-Objectives and scope
+Objectives and Scope
 ++++++++++++++++++++
 
+Change Management Goals
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* Requests for Changes are recorded and identified.
+* Change Requests are analyzed, dependencies and relationships to Work Products or other Change
+  Requests are identified, and the impact is estimated.
+* Change Requests are approved before implementation and prioritized accordingly.
+* Bidirectional traceability is established between Change Requests and affected work products.
+* Implementation of Change Requests is confirmed.
+* Change Requests are tracked to closure and status of Change Requests is communicated to
+  affected parties.
+* Requests for Changes are properly documented.
+
 Approach
 ++++++++
+
+Change Request Execution
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Contributions in general to the **S-CORE** project are described here
+(compare :need:`gd_guidl__contr_request_guideline`).
+
+A Change Request is a specific contribution, and
+it is the **ONLY** way to contribute new features or to modify the scope of existing features
+in the **S-CORE** project.
+
+Change Request Infrastructure and Types
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:need:`GitHub Issues (ISSUE) <gd_guidl__issue_guideline>` are used for managing Change Requests.
+The tool is used to create, plan, control, and monitor Change Requests within **S-CORE**.
+
+:need:`GitHub Pull Requests (PR) <gd_guidl__pull_request_guideline>` are used for the implementation
+of Change Requests. The tool is used to implement and verify Change Requests within **S-CORE**.
+
+The next figure gives an overview, how Change Requests are realized in **S-CORE**. An ISSUE is
+used to create a Change Request including reuqired attributes as defined in
+:ref:`chm_process_change_request_attributes`.
+The ISSUE may be linked to SUB-ISSUEs, if required, to manage more complex Change Requests.
+Implementation of Change Requests requires at least one or more PRs linked to the ISSUE created
```

Comment:
"At least one" already implicates that it can be more.

```suggestion
The implementation of a Change Requests requires at least one PR linked to the ISSUE created
```